packages/orchestration/src/types.ts

```diff
@@ -18,7 +18,23 @@ export type * from './vat-orchestration.js';
 /**
  * Partial Zoe Contract Facet ({@link ZCF}) for use in {@link OrchestrationFlow}s.
  *
- * @see {@link HostOf} for the analogy between `Promise` and `Vow`.
  * @interface
  */
-export type ZcfForFlows = ReturnType<typeof prepareZcfForFlows>;
+export type ZcfForFlows<CT = Record<string, unknown>> = Pick<
+  ZCF<CT>,
+  'atomicRearrange' | 'assertUniqueKeyword' | 'getTerms'
+> & {
+  makeZCFMint: <K_2 extends AssetKind = 'nat'>(
```

Comment:
`K_2` appears to be a vestige of a state that had `K`.
```suggestion
  makeZCFMint: <K extends AssetKind = 'nat'>(
```